ui/lib/analysis/expired-jwt.ts

```diff
@@ -0,0 +1,106 @@
+import { Analysis, ExpiredJWT } from '../findings';
+import conn from '../db';
+import jwtDecode, { JwtPayload } from 'jwt-decode';
+
+const query = `
+SELECT
+  jsonb_array_elements((meta->>'DetectedJwts')::jsonb) as jwt,
+  data->'request'->>'absoluteURI' as absolute_uri,
+  data->'protocol'->>'name' as proto,
+  data->'src'->>'ip' as src_ip,
+  data->'src'->>'port' as src_port,
+  data->'dst'->>'ip' as destination_ip,
+  data->'dst'->>'port' as destination_port,
+  data->'timestamp' as timestamp
+  FROM traffic WHERE
+  meta->>'DetectedJwts' is not null;
+`;
+
+interface Row {
+  jwt: string;
+  absolute_uri: string;
+  protocol: string;
+  src_ip: string;
+  src_port: string;
+  destination_ip: string;
+  destination_port: string;
+  timestamp: number;
+}
+
+function isExpired(jwt: string, occuredAt: number): string {
```

Comment:
Indicate missing values with explicit undefined, rather than empty string:
```suggestion
function isExpired(jwt: string, occuredAt: number): string | undefined {
```